lib/widgets/settings_listing.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:flutter/material.dart';
+import 'package:badiup/constants.dart' as constants;
+import '../colors.dart';
+import '../sign_in.dart';
+
+class SettingsListing extends StatefulWidget {
+  @override
+  _SettingsListingState createState() => _SettingsListingState();
+}
+
+class _SettingsListingState extends State<SettingsListing> {
+  bool _value = currentSignedInUser.setting.pushNotifications;
```

Comment:
use `_ifPushNotifications` or `pushNotificationsBoolValue`, ( or other easier understanding name ) instead of `_value`.